src/cli/commands/test/cache.rs

```diff
@@ -0,0 +1,31 @@
+pub mod cache {
+	use std::{fmt::Debug, fs::read_to_string, path::PathBuf};
+
+	use thiserror::Error;
+
+	use serde::{Deserialize, Serialize};
+	use serde_json;
+
+	#[derive(Error, Debug)]
+	pub enum CacheError {
+		#[error("file `{0}` not found ")]
+		FileNotFound(PathBuf),
+		#[error("failed to deserialize file '{0}': {1}")]
+		DeserializeError(String, serde_json::Error),
+	}
+
+	#[derive(Serialize, Deserialize, Debug, PartialEq)]
+	pub struct CacheCairoFoundry {
```

Comment:
I think you can probably call it `Cache` here as we know this is the CairoFoundry project

Reply:
gotcha!